src/stlink-gui/CMakeLists.txt

```diff
@@ -13,11 +13,11 @@ if (NOT WIN32)
 
         # Install desktop application entry
         install(FILES stlink-gui.desktop
-                DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/${PROJECT_NAME}/applications)
+                DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/applications)
 
         # Install icons
         install(FILES icons/stlink-gui.svg
-                DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/${PROJECT_NAME}/icons/hicolor/scalable/apps)
+                DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/icons/hicolor/scalable/apps)
```

Comment:
I'd set the second path to `/usr/share/icons/stlink/...` to avoid a mixture of files with other applications within that subdirectory. Can you update that please?

Reply:
Why not mixture icons with other applications? All icons should be in one place so in desktop-file you can just write `stlink.svg` instead of full absolute path. It will work in any system.

Reply:
If this is the only way to achieve this - ok. I just want to make sure we don't run into any clutter when users need to locate files during a manual clean-up without any harm to other applications. That was the main point.

Reply:
It is best way.
```
ls /usr/share/icons/hicolor/scalable/apps/ | wc -l
83
```
on my system.